wgengine/monitor/monitor.go

```diff
@@ -52,7 +52,7 @@ type osMon interface {
 // changed. The changed parameter is whether the network changed
 // enough for interfaces.State to have changed since the last
 // callback.
-type ChangeFunc func(changed bool, state *interfaces.State)
+type ChangeFunc = func(changed bool, state *interfaces.State)
```

Comment:
Had to change this to a type alias since otherwise I was getting complaints that `Mon` was not satisfying the `LinkMonitor` interface.

Reply:
Type aliases are kinda gross. They should only be temporary during multistep migrations.

Reply:
You can put the ChangeFunc type in the interfaces package by type State.

Reply:
Sounds good, moved it to `interfaces.go`.